code/datums/ai/monkey/monkey_behaviors.dm

```diff
@@ -25,6 +25,10 @@
 		finish_action(controller, FALSE)
 		return
 
+	if(HAS_TRAIT(target, TRAIT_NEEDS_TWO_HANDS)) //Can't carry items that need two hands as a monkey
```

Comment:
This should be done in the ai controller so it doesn't bother going for weapons it cant go for.

Reply:
I have moved this to the controller side.